update HTTP/3 docs to reflect RFC9000 and UA changes (#2181)
cloudflare/cloudflare-docs: products/http3/src/content/index.md

The Hypertext Transfer Protocol (HTTP) is used by clients such as web browsers to talk to servers in order to load websites. The performance of HTTP is an important factor when it comes to loading web pages quickly and efficiently.

> HTTP is a well established protocol that has several versions, with each adding

Reviewer: ```suggestion
HTTP is a well established protocol that has several versions, and each version
```

> features that improve performance over the older one. HTTP/1.1 and HTTP/2 are

Reviewer: ```suggestion
adds features that improve performance over the older one. HTTP/1.1 and HTTP/2 are
```

widely deployed on the Internet today and rely on TCP and optionally TLS.

[HTTP/3](https://blog.cloudflare.com/http3-the-past-present-and-future/) is the
latest version that runs over the new transport protocol
[QUIC](https://blog.cloudflare.com/the-road-to-quic/). [QUIC version 1](https://blog.cloudflare.com/quic-version-1-is-live-on-cloudflare/)

> is published as RFC 9000 and is supported by Cloudflare, HTTP/3 connections using

Reviewer: ```suggestion
is published as RFC 9000 and is supported by Cloudflare. HTTP/3 connections using
```

## Supported clients

The following clients support HTTP/3:

- [Google Chrome](https://www.google.com/chrome/canary/) - [Instructions](/chrome)
- [Microsoft Edge](https://www.microsoftedgeinsider.com/en-us/) - [Instructions](/edge)
- [Mozilla Firefox](https://www.mozilla.org/firefox/channel/desktop/#nightly) - [Instructions](/firefox)
- [cURL](https://curl.haxx.se) + quiche - [Instructions](/curl-brew)
- [Cloudflare quiche-client](https://github.com/cloudflare/quiche) - [Instructions](/quiche-http3-client)

<Aside>

> __Note:__ Cloudflare supports multiple draft versions of HTTP/3 and QUIC. When using a client, please check that it supports one of the versions  listed in the [release notes](/release).

Reviewer: ```suggestion
__Note:__ Cloudflare supports multiple draft versions of HTTP/3 and QUIC. When using a client, verify that it supports one of the versions listed in the [release notes](/release).
```
